src/Microsoft.Framework.Caching.SqlServer/SqlQueries.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Framework.Caching.SqlServer
+{
+    internal class SqlQueries
+    {
+        private const string CreateTableFormat = "CREATE TABLE [{0}](" +
```

Comment:
Don't we want to support schemas with the same table name, e.g. for multi-tenancy? In that case you will have to do something about the square brackets, e.g. [schemaname].[tablename] is ok, but I believe that [schemanam.tablename] is not.